demo/index.html

```diff
@@ -10,13 +10,16 @@
   <body>
     <main></main>
 
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/prismjs/prism.js"></script>
+
     <script type="module">
-      import "https://unpkg.com/marked@latest/marked.min.js";
-      import "https://unpkg.com/prismjs@latest/prism.js";
+      // import "https://unpkg.com/marked@latest/marked.min.js";
+      // import "https://unpkg.com/prismjs@latest/prism.js";
```

Comment:
Please see https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/demo/index.html for the correct configuration. 

Also, please remove commented code when prepared for PR review.